modal/_utils/pattern_matcher.py

```diff
@@ -151,44 +152,43 @@ def match(self, path: str) -> bool:
 class PatternMatcher:
```

Comment:
Since we might want this as a top level `modal.X` entity, we might have to think of a name that is less general and conveys that it's matching paths specifically. Naming is hard though... What about `MatchingFiles`?

Reply:
How about the `noun` `FilePatternMatcher` ? 😄